docs/apis-tools/zeebe-api/gateway-service.md

```diff
@@ -858,9 +858,11 @@ Returned if:
 - Not all active elements in the given process instance are mapped to the elements in the target process definition
 - A mapping instruction changes the type of an element or event
 - A mapping instruction changes the implementation of a task
+- A mapping instruction detaches a boundary event from an active element
```

Comment:
❌ We still need adjust this to cover all catch events
```suggestion
- A mapping instruction detaches a catch event from an active element
```

Reply:
@korthout this limitation was outdated as I added an example after it.  I refactored it to catch events in general but keep the example for boundary event. Please take another look. 👀